modules/cloud-library/connect/cloud-library.php

```diff
@@ -124,5 +126,23 @@ public function update_resource( array $template_data ) {
 		return true;
 	}
 
+	public function get_connect_url() {
+		if ( Utils::has_pro() ) {
+			return Plugin::$instance->common->get_component( 'connect' )->get_app( 'activate' )->get_admin_url( 'authorize', [
+				'utm_source' => 'template-library',
+				'utm_medium' => 'wp-dash',
+				'utm_campaign' => 'connect-and-activate-license',
+				'utm_content' => 'cloud-library',
+			] );
+		}
+
+		return Plugin::$instance->common->get_component( 'connect' )->get_app( 'library' )->get_admin_url( 'authorize', [
+			'utm_source' => 'template-library',
+			'utm_medium' => 'wp-dash',
+			'utm_campaign' => 'library-connect',
+			'utm_content' => 'cloud-library',
+		] );
```

Comment:
You can connect to the Library today even without Pro. In this case, the user is "connected" but then we will probably just check the quota.

Reply:
I'm actually not 100% sure about this part. I am calling code from Pro here 🤔